boards/riscv/niosv_m/doc/index.rst

```diff
@@ -0,0 +1,71 @@
+.. _niosv_m:
+
+INTEL FPGA niosv_m
+####################
+
+Overview
+********
+
+niosv_m board is based on Intel FPGA Design Store Nios® V/m Hello World Example Design system and this complete system is consisted of following IP blocks:
+
+.. code-block:: console
+
+	Nios® V/m Processor Intel® FPGA IP
+	JTAG UART Intel® FPGA IP
+	On-Chip Memory Intel® FPGA IP
+
+Nios® V/m hello world example design system
+===========================================
+
+Prebuilt Nios® V/m hello world example design system is available in Intel FPGA Design store.
+- https://www.intel.com/content/www/us/en/support/programmable/support-resources/design-examples/design-store.html?s=Newest
+
+.. code-block:: console
```

Comment:
Why these empty code blocks? Maybe you want to check how the generated docs look like.

Reply:
The empty code blocks have been removed. Thanks.